iFixit/react-commerce
# [Feat] Product: Add polaroid genuine part svg

Closes https://github.com/iFixit/ifixit/issues/51727

## QA

Find (or create) a product that has the "OEM Partnership" metafield set to Polaroid in iFixit Test Shopify Admin. It should be a json object like:

```
{ code: "polaroid", text: "This is a genuine Polaroid product.", url: "https://www.polaroid.com/en_us" }
```

And it should look something like this:

![image](https://github.com/iFixit/react-commerce/assets/52104630/62316e59-6e26-4684-bc5b-2b9d1ef2628d)
